helper.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import argparse
+from glob import iglob
+
+
+def create_output():
+    """Create single JSON object from found JSON files"""
+    import json
+
+    results = {}
+    for filename in iglob("*.json"):
+        name = filename[:-len(".json")]
+        with open(filename) as handle:
+            try:
+                results[name] = json.load(handle)
+            except json.JSONDecodeError:
+                continue
+    print(json.dumps(results))
+
+
+def delete_files(path):
+    """Delete files"""
+    import os
+
+    for filename in iglob(path):
+        os.remove(filename)
```

Comment:
True. Since the point of the function here is to _only_ deal with files, it should maybe even raise if it comes upon a folder though? But maybe doing this is more practical.